### ISSUE TITLE
Siamese Network problem

### ISSUE THREAD

**Issue by dongguangyao** [BVLC/caffe#1425]
Dear Sir:
  I am wondering how to achieve face verification based on caffe. Because it pre-process the images and convert them into lmdb or leveldb. Therefore, if I utilize two images as input and set the target value(same person). How could I set the target, all belong to 1, However, if the the first person was compared with itself, the target should be 1. In contract, if he was compared with the other persons?
  In conclusion, how to set the target?
